students/nartowska_karina/lesson_05_lists/age_calculator.py

```diff
@@ -0,0 +1,41 @@
+def average_age(values):
+    average = 0
+    adult = 0
+    for i in values:
+        if i >= 18:
+            average += i
+            adult += 1
+    if adult > 0:
+        print("The average age of adults: ")
+        return average / adult
+    else:
+        print("The average age of adults: ")
+        return 0
+
+
+def count_the_children(values):
+    children = 0
+    for i in values:
+        if i < 18:
+            children += 1
+    print("Count the children: ")
```

Comment:
Same as previously. You are printing some information string and after then you print result in the main function which is connected with this previously string - it is a bit confusing. Please avoid similar situations in next lessons :)